packages/shared/src/server/queues.ts

```diff
@@ -71,6 +71,10 @@ export const TracesQueueEventSchema = z.object({
   projectId: z.string(),
   traceIds: z.array(z.string()),
 });
+export const ProjectQueueEventSchema = z.object({
```

Comment:
Lets rename this to ProjectDeleteQueueEventSchema

Reply:
I actually like it that way because we could reuse it for project create events or similar ones. There is no reason to tie the body of a projectId/orgId combination to delete events in my mind.